test/unit/ObjectMakrTests/getFullProperties.js

```diff
@@ -0,0 +1,58 @@
+define(["mocks"], function (mocks) {
+    return function () {
+        var expect = require("chai").expect;
+
+        it("returns the full properties", function () {
+            // Arrange
+            var objectMaker = mocks.mockObjectMakr();
+
+            var fullProperties = {
+                    "Animal": {
```

Comment:
Unless they're necessary, can you leave out quotes in keys? I'm trying to change the style in all FSS modules.